tests/mocha.opts

```diff
@@ -1,2 +1,3 @@
 --colors
+--compilers js:pouchdb-plugin-helper/mocha-babel 
```

Comment:
I would much prefer to keep things simple and not have any ES6+ in the codebase, but if that's how pouchdb-replicator and the other plugins are written, then we should probably have a unified strategy for what we're doing with transpilation. It seems ES6+ is currently only used in the tests, not the source code?